markets/perps-market/storage.dump.sol

```diff
@@ -50,11 +50,11 @@ library ERC2771Context {
 
 // @custom:artifact @synthetixio/core-contracts/contracts/utils/HeapUtil.sol:HeapUtil
 library HeapUtil {
-    uint private constant _ROOT_INDEX = 1;
```

Comment:
i dont know why storage dump changed, it might just have not been committed in a while

Reply:
This is due to the explicit types new rule in the repo (uint/int should be explicit uint256/int256). A PRs added that a couple of days ago.
Storage allocation should be the same.